docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -179,6 +179,9 @@ How many rows are fetched at a time by the JDBC driver.
 `quarkus.hibernate-orm.jdbc.statement-batch-size`::
 The number of updates (inserts, updates and deletes) that are sent by the JDBC driver at one time for execution.
 
+`quarkus.hibernate-orm.globally-quoted-identifiers`:: (defaults to `false`)
```

Comment:
The property key is incorrect here as it has been added in the `jdbc` section.

Reply:
Fixed, thanks.